### PR TITLE
Implement `--meta_input` with `post create` & `post update`

### PR DESCRIPTION
**Pull request by schlessera**
This needs https://github.com/wp-cli/wp-cli/pull/4623 to land first.

See https://github.com/wp-cli/wp-cli/pull/4616

### CODE REVIEW

File: src/Post_Command.php [hunk 1]
```diff
@@ -143,6 +143,10 @@ public function __construct() {
 	 *     # Create post with content from given file
 	 *     $ wp post create ./post-content.txt --post_category=201,345 --post_title='Post from file'
 	 *     Success: Created post 1922.
+	 *
+	 *     # Create a post with multiple meta values.
+	 *     $ wp post create --post_title='A post' --post_content='Just a small post.' --meta_input='{"key1":"value1","key2":"value2"}
```
Comment (Mr-Alexander) [inline src/Post_Command.php]: The `'` is not balanced here.

`--meta_input='{"key1":"value1","key2":"value2"}'`

File: src/Post_Command.php [hunk 2]
```diff
@@ -271,6 +275,10 @@ public function create( $args, $assoc_args ) {
 	 *
 	 *     $ wp post update 123 --post_name=something --post_status=draft
 	 *     Success: Updated post 123.
+	 *
+	 *     # Update a post with multiple meta values.
+	 *     $ wp post update 123 --meta_input='{"key1":"value1","key2":"value2"}
```
Comment (Mr-Alexander) [inline src/Post_Command.php]: The ' is not balanced here.

it should be :
`--meta_input='{"key1":"value1","key2":"value2"}'`

Comment (swissspidy) [reply src/Post_Command.php]: Good catch, thanks!

File: src/Post_Command.php (hunk 1 above)
Comment (swissspidy) [reply src/Post_Command.php]: This has been fixed a while ago in #147.

**Pull request merged**